arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -240,6 +240,13 @@ bool VstCommTask<T>::processMessage(velocypack::Buffer<uint8_t> buffer,
       this->_auth->userManager()->refreshUser(this->_authToken._username);
     }
     
+    LOG_TOPIC("92fd6", INFO, Logger::REQUESTS)
```

Comment:
```suggestion
    LOG_TOPIC("92fd6", DEBUG, Logger::REQUESTS)
```
same here

Reply:
REQUESTS log topic is *FATAL* by default, this is fine